addons/website/views/snippets/s_text_cover.xml

```diff
@@ -2,15 +2,17 @@
 <odoo>
 
 <template id="s_text_cover" name="Text Cover">
-    <section class="s_text_cover">
+    <section class="s_text_cover o_cc o_cc2">
         <div class="container-fluid">
-            <div class="row">
-                <div class="o_colored_level o_cc o_cc4 col-lg-5 pt160 pb160 px-5">
+            <div class="row o_grid_mode" data-row-count="11">
+                <div class="o_grid_item g-height-10 g-col-lg-5 col-lg-5 o_colored_level o_cc o_cc1" style="z-index: 1; grid-area: 2 / 3 / 11 / 8; --grid-item-padding-x: 24px; --grid-item-padding-y: 24px;">
                     <h1 class="display-3">Sell Online. <br/>Easily.</h1>
-                    <p class="lead">Write one or two paragraphs describing your product, services or a specific feature. To be successful your content needs to be useful to your readers.</p>
-                    <a t-att-href="cta_btn_href" class="btn btn-primary"><t t-esc="cta_btn_text">Contact us</t></a>
+                    <p class="lead"><br/>Sell online easily with a user-friendly platform that streamlines all the steps, including setup, inventory management, and payment processing.<br/></p>
+                    <a t-att-href="cta_btn_href" class="btn btn-lg btn-primary"><t t-esc="cta_btn_text">Contact us</t></a>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-height-12 g-col-lg-6 col-lg-6 o_not_editable o_colored_level o_cc o_cc1 pt160 pb160 oe_img_bg d-none d-md-block" style="grid-area: 1 / 7 / 12 / -1; --grid-item-padding-x: 0px; --grid-item-padding-y: 0px;">
+                    <img class="img img-fluid mx-auto" src="/web/image/website.s_text_cover_default_image"/>
```

Comment:
It was already the case before - but maybe less obvious given that it was used as a background image: the image is not replaced by industry when using the configurator. A fallback can be defined which will be used until industry images are specified. See <https://github.com/odoo/odoo/blob/master/addons/website/models/website.py#L851>.

Reply:
Not sure to understand this, does it mean I should simply add this ?  `fallback_create_missing_industry_image('s_text_cover_default_image, 's_cover_default_image')`

It doesn't look like something we do for all the snippets so I'm not sure it's the right place 😅

Reply:
Yes, I think we should add similar lines in that list whenever we find images for which no mapping per industry has been defined yet. Ultimately someone might define specific images - when that will happen, that fallback will just be ignored.
I leave it to you to chose what replacement image should be used.